tests/testthat/test-glue.R

```diff
@@ -267,8 +267,15 @@ glue("
 
 test_that("glue works with alternative delimiters", {
   expect_identical(as_glue("{1}"), glue("{1}", .open = "", .close = ""))
+  expect_identical(as_glue("{{}}"), glue("{{}}", .open = "", .close = ""))
 
   expect_identical(as_glue("1"), glue("<<1>>", .open = "<<", .close = ">>"))
+  expect_identical(as_glue("<<<<>>>>"), glue("<<<<>>>>", .open = "<<", .close = ">>"))
```

Comment:
This test is wrong, doubled delimiters are converted to singular in the output, so it should be testing against `as_glue("<<>>")`